mathicsscript/bindkeys.py

```diff
@@ -16,45 +16,97 @@
 from typing import Callable
 from prompt_toolkit.enums import EditingMode
 from prompt_toolkit.key_binding import KeyBindings
-
+from prompt_toolkit.filters import Condition
 import pathlib
 import re
 
+from mathicsscript.settings import definitions
+from mathics.session import get_settings_value
+
+
+@Condition
+def autocomplete_on():
+    return get_settings_value(definitions, "Settings`$GroupAutocomplete")
```

Comment:
Mention of this should be added to `mathicsscript/mathicsscript/user-settings.m`

Also please add a `Settings`$PreferredBackendMethod::usage="..."` assignment so that users will know inside mathicsscript what this does. Thanks.